prettier.config.cjs

```diff
@@ -3,7 +3,7 @@ module.exports = {
   bracketSpacing: true,
   htmlWhitespaceSensitivity: 'css',
   insertPragma: false,
-  jsxBracketSameLine: false,
+  bracketSameLine: false,
```

Comment:
See deprecation and renaming of property here: https://prettier.io/docs/en/options.html#deprecated-jsx-brackets